spec/lib/spree/adyen/invoice_spec.rb

```diff
@@ -6,7 +6,7 @@
     let(:tax_rate_country) { address.country }
     let(:tax_category) { create(:tax_category) }
     let!(:zone) { create(:zone, name: "Country Zone", default_tax: true, countries: [tax_rate_country]) }
-    let!(:rate) { create(:tax_rate, tax_categories: [tax_category], amount: 0.19, included_in_price: true, zone: zone) }
+    let!(:rate) { create(:tax_rate, tax_category: tax_category, amount: 0.19, included_in_price: true, zone: zone) }
```

Comment:
are you sure this is actually updated? Tax rates now have multiple tax categories